swagger.yaml

```diff
@@ -299,6 +299,49 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/supported-platforms:
+    get:
+      tags:
+        - installer
+      description: Get all cluster supported platforms
```

Comment:
Get cluster supported platforms

Reply:
Can we add a succinct description of how this is determined?